documentation/create_overview_doc/README.md

```diff
@@ -51,21 +51,27 @@ sh
 kt
 ```
 
+For flow map generation with Code2Flow, the tool currently supports the following languages, each with specific dependencies:
+
+- JavaScript: Requires Acorn for parsing. Ensure Acorn is installed and accessible in your environment.
+- Ruby: Uses Parser for parsing. Ensure the Parser gem is installed and accessible.
+- PHP: Employs PHP-Parser for parsing. PHP-Parser should be installed and accessible in your environment.
```

Comment:
**Performance Issue**: The addition of language-specific dependencies for Code2Flow might introduce performance bottlenecks if these dependencies are not optimized or if they introduce significant overhead during the flow map generation process. <br> **Fix**: Ensure that the dependencies such as Acorn for JavaScript, Parser gem for Ruby, and PHP-Parser are optimized for performance. Consider lazy loading these dependencies or using alternative lightweight libraries if performance issues are detected. <br> **Code Suggestion**: 
 ```
 +Ensure that the dependencies such as Acorn for JavaScript, Parser gem for Ruby, and PHP-Parser are optimized for performance. Consider lazy loading these dependencies or using alternative lightweight libraries if performance issues are detected.
 ```